packages/expo-local-authentication/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### 🐛 Bug fixes
 
+- Added docs about Android permissions. ([#9492](https://github.com/expo/expo/pull/9492) by [@bycedric](https://github.com/bycedric))
```

Comment:
Hmm, docs changes are not user-facing so I think there is no benefit from adding such changelog entries, especially that it's misleading to say it is a bug fix 😅 Anyway, I'm fine with leaving it as is.

Reply:
Good point, I'll remove it here and in the other PRs as well!